src/ui/router/ScrollToTop.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function ScrollToTop() {
```

Comment:
Already asked above, but why is this treated as a non-rendering component instead of a "custom hook"